i18n/update_translations.py

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env python
```

Comment:
I still believe that this script isn't something anyone would write in JS. However, I also don't know enough about JS (command line args, parsing json, calling system commands etc.) to easily translate the script to JS.